meson.build

```diff
@@ -154,6 +154,17 @@ if not get_option('cairo').disabled()
   endif
 endif
 
+chafa_dep = null_dep
+if not get_option('chafa').disabled()
+  chafa_dep = dependency('chafa', version: '>= 1.6.0', required: false)
+
+  if (not chafa_dep.found() and
+      cpp.get_id() == 'msvc' and
+      cpp.has_header('chafa.h'))
+    chafa_dep = cpp.find_library('chafa', required: false)
+  endif
```

Comment:
I’d not bother with MSVC here, and simplify this into:
```
chafa_dep = dependency('chafa', version: '>= 1.6.0', required: get_option('chafa'))
```

Reply:
Done!